jwt.go

```diff
@@ -594,15 +597,22 @@ func (jwtPlugin *JwtPlugin) CheckOpa(request *http.Request, token *JWT, rw http.
 	if err = json.Unmarshal(fieldResult, &allow); err != nil {
 		return 0, err
 	}
+
 	if !allow {
+		var notAllowErr error
+		if jwtPlugin.opaDebugMode {
+			notAllowErr = fmt.Errorf("%s", body)
+		} else {
+			notAllowErr = fmt.Errorf("forbidden")
```

Comment:
there is no need to format, you should return constant string.

Reply:
this project uses the fmt.Errorf even when returning a constant string as far as I can tell

Reply:
The alternative would be to use an error wrapper interface, not sure if it's really worth all that extra effort, to be honest....